config/rbac/role.yaml

```diff
@@ -4,6 +4,35 @@ kind: ClusterRole
 metadata:
   name: nimbus-operator
 rules:
+- apiGroups:
```

Comment:
Do you need changes in this file ? For this intent, we do not need any additional permissions ..